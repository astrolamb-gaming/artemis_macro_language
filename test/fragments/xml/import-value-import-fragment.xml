<mission_data version="1.0">
    <mission_description>  Multiplayer mission with Jump drive ships. Capture the most eggs by towing them to your station. Eggs can only be towed at impulse. Comms Button at start of mission must be pressed to join game. </mission_description>
    <imports>
        <import name="import-value-import-value.xml" />
        <import name="import-value-import-${indirect}.xml" />
        <repeat _range="myrange"  >	
            <import name="import-value-import-${file}.xml" />
        </repeat>
         <repeat _range="myrange"  _as="O">	
            <repeat _range="myrange"  _as="I">	
                <import name="import-${I.v}-${O.imp}-${I.file}.xml" />
            </repeat>
        </repeat>
        <import name="import-value-import-${indirect}.xml" />
    </imports>
</mission_data> 