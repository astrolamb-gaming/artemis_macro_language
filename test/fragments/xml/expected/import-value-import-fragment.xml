<mission_data version="1.0">
  <mission_description>  Multiplayer mission with Jump drive ships. Capture the most eggs by towing them to your station. Eggs can only be towed at impulse. Comms Button at start of mission must be pressed to join game. </mission_description>
  <start>
    <big_message title="Indirect" subtitle1="Import"/>
    <big_message title="Indirect" subtitle1="Import"/>
    <big_message title="Indirect" subtitle1="Import1"/>
    <big_message title="Indirect" subtitle1="Import"/>
  </start>
</mission_data>